tell/match.py

```diff
@@ -220,7 +226,7 @@ def filter_one(df_fips, df_states, df_ult, df_ba):
     df_fips_ult = df_states_fips.merge(df_ult, on='Utility Number', how='left')
 
     # add the BA number and name by BA ulitity code
-    df_valid = df_fips_ult.merge(df_ba, on='BA Code', how='left')
+    df_valid = df_fips_ult.merge(df_ba, left_on='BA Code', right_on='BA Code', how='left')
```

Comment:
whoops, this change isn't necessary... must have been something i was experimenting with